File: src/main/Java/boj_step13/boj2587.java
import java.io.*;
import java.util.Arrays;

public class boj2587 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        int[] list = new int[5];
        for (int i = 0; i < list.length; i++) list[i] = Integer.parseInt(reader.readLine());
        list = Arrays.stream(list).sorted().toArray();
        writer.append(String.format("%.0f", Arrays.stream(list).average().getAsDouble()));
        writer.newLine();
        writer.append(String.valueOf(list[2])).close();
    }
}

File: src/main/boj_step13/boj2587.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    var list = mutableListOf<Int>()
    repeat(5) {
        list.add(r.readLine().toInt())
    }
    w.write("${list.sum() / list.size}\n")
    list = list.sorted().toMutableList()
    w.write(list[2].toString())
    w.close()
}